### PR TITLE
Fix(cv_deploy): Ensure 'AVD Configurations' container is inserted first

### PR DESCRIPTION
**Pull request by ClausHolbechArista**
## Change Summary

Ensure 'AVD Configurations' container is inserted first so configs can be overridden by other static configlets or reconciled configs.

## Related Issue(s)

BUG990962

## Component(s) name

`arista.avd.cv_deploy`

## How to test

Tested manually with `cv_deploy` molecule scenario. Added a reconcile container and deleted the AVD config container. The container was recreated and inserted first.

### User Checklist

- N/A

### Repository Checklist

- [x] My code has been rebased from devel before I start
- [x] I have read the [**CONTRIBUTING**](https://avd.sh/en/latest/docs/contribution/overview.html) document.
- [x] My change requires a change to the documentation and documentation have been updated accordingly.
- [x] I have updated [molecule CI](https://github.com/aristanetworks/avd/tree/devel/ansible_collections/arista/avd/molecule) testing accordingly. (check the box if not applicable)

### CODE REVIEW

File: python-avd/pyavd/_cv/workflows/deploy_configs_to_cv.py [hunk 1]
```diff
@@ -111,7 +111,8 @@ async def get_existing_device_container_ids_from_root_container(workspace_id: st
     LOGGER.info("deploy_configs_to_cv: Found %s root containers.", len(root_containers))
     if CONFIGLET_CONTAINER_ID not in root_containers:
         LOGGER.info("deploy_configs_to_cv: AVD root container not assigned as root container in Static Config Studio. Fixing...")
-        root_containers.append(CONFIGLET_CONTAINER_ID)
+        # Inserting our container first, to allow reconcile and other static config containers to override the AVD config.
```
Comment (carl-baillargeon) [inline python-avd/pyavd/_cv/workflows/deploy_configs_to_cv.py:114]: I would update the log message above to clarify this.

Comment (ClausHolbechArista) [reply python-avd/pyavd/_cv/workflows/deploy_configs_to_cv.py:114]: The log msg should not explain the full behavior of the code and background. But I can improve the message a bit.

Comment (ClausHolbechArista) [reply python-avd/pyavd/_cv/workflows/deploy_configs_to_cv.py:114]: updated

**Review by carl-baillargeon — APPROVED** [review]
LGTM

**Review by carlbuchmann — APPROVED** [review]
LGTM

**Pull request merged**